kivy/uix/textinput.py

```diff
@@ -2583,6 +2585,15 @@ def keyboard_on_textinput(self, window, text):
         if self._selection:
             self.delete_selection()
         self.insert_text(text, False)
+        self._is_textedit=False #准备插入，对束edit状态
```

Comment:
comment should be in english. also use space around `=`.
```suggestion
        self._is_textedit = False  # Ready to insert, on bundle edit status
```
(comment translated by google, not sure of the meaning, if needed, please clarify).

Reply:
it is '结束',not '对束'.
I have modifeid like this:

 self._is_textedit = False #ready to insert ,finish text edit state.